src/Rector/Contrib/PHPUnit/SpecificMethodRector.php

```diff
@@ -104,9 +103,11 @@ private function renameMethod(MethodCall $methodCallNode): void
         [$trueMethodName, $falseMethodName] = $this->oldToNewMethods[$this->activeFuncCallName];
 
         if ($oldMethodName === 'assertTrue' && $trueMethodName) {
-            $methodCallNode->name = $trueMethodName;
+            /** @var string $trueMethodName */
+            $methodCallNode->name = new Identifier($trueMethodName);
```

Comment:
This is not related, just best practise. In PHP-Praser the string would be converted to `Identifier` anyway, this makes it explicit.